packages/fxa-settings/src/components/ConnectAnotherDevicePromo/en-US.ftl

```diff
@@ -4,4 +4,11 @@ connect-another-fx-mobile = Get { -brand-firefox } on mobile or tablet
 connect-another-find-fx-mobile = Find { -brand-firefox } in the { -google-play } and { -app-store } or
   <br /><linkExternal>send a download link to your device.</linkExternal>
 
+# Alt text for Google Play and Apple App store images that will be shown if the image can't be loaded.
+# These images are used to encourage users to download Firefox on their mobile devices.
+connect-another-play-store-image =
+    .title = Download { -brand-firefox } for Android
```

Comment:
I mentioned this in Slack but just bringing it up here too since I'm r+'ing:

We should add "Android" and "iOS" to `App/en-US.ftl` as a brand reference, see [the docs](https://projectfluent.org/fluent/guide/references.html) for more info.